src/include/checkpoint.h

```diff
@@ -9,24 +9,22 @@
 #define WT_CHECKPOINT_SESSION_FLAGS (WT_SESSION_CAN_WAIT | WT_SESSION_IGNORE_CACHE_SIZE)
 
 #define WT_CHECKPOINT_STATE_INACTIVE 0
-#define WT_CHECKPOINT_STATE_RESET_CURSORS 1
-#define WT_CHECKPOINT_STATE_FLUSH_TIER_WAIT 2
-#define WT_CHECKPOINT_STATE_APPLY_META 3
-#define WT_CHECKPOINT_STATE_APPLY_BTREE 4
-#define WT_CHECKPOINT_STATE_UPDATE_OLDEST 5
-#define WT_CHECKPOINT_STATE_SYNC_FILE 6
-#define WT_CHECKPOINT_STATE_EVICT_FILE 7
-#define WT_CHECKPOINT_STATE_BM_SYNC 8
-#define WT_CHECKPOINT_STATE_RESOLVE 9
-#define WT_CHECKPOINT_STATE_POSTPROCESS 10
-#define WT_CHECKPOINT_STATE_HS 11
-#define WT_CHECKPOINT_STATE_HS_SYNC 12
-#define WT_CHECKPOINT_STATE_COMMIT 13
-#define WT_CHECKPOINT_STATE_META_CKPT 14
-#define WT_CHECKPOINT_STATE_META_SYNC 15
-#define WT_CHECKPOINT_STATE_ROLLBACK 16
-#define WT_CHECKPOINT_STATE_LOG 17
-#define WT_CHECKPOINT_STATE_RUNNING 18
-#define WT_CHECKPOINT_STATE_ESTABLISH 19
-#define WT_CHECKPOINT_STATE_START_TXN 20
-#define WT_CHECKPOINT_STATE_CKPT_TREE 21
+#define WT_CHECKPOINT_STATE_APPLY_META 1
```

Comment:
Usually, our macros are in alphabetical order. Is it possible to rearrange it here also?

Reply:
Created [WT-12390](https://jira.mongodb.org/browse/WT-12390) to deal with this as we need a long term solution.